test/utils/utils.go

```diff
@@ -24,3 +30,27 @@ func SpecDescription(spec string) string {
 	}
 	return spec
 }
+
+//OperatorIsHiveManaged Returns true if addon is manged by hive
+func OperatorIsHiveManaged(client k8sclient.Client, inst *v1alpha1.RHMI) (bool, error) {
```

Comment:
hey @Boomatang can we use the existing function for this? https://github.com/integr8ly/integreatly-operator/blob/master/pkg/addon/operator_run.go#L126

Reply:
While that is something that could be done it is not a good practices to depend on the code that is under test as part of a black box test. Testing code and production code should have a degree of separation.

Reply:
While I would agree to some degree, I think this is different. 
The function is very simple and does not require to be modified for the purpose of this test, additionally, if we would ever have to change the way of figuring out if a cluster is hive managed or not, this change would require updates in 2 places. I think we should avoid duplication in our code as much as we can.